tachyon/zk/air/constraint_system/BUILD.bazel

```diff
@@ -1,7 +1,18 @@
-load("//bazel:tachyon_cc.bzl", "tachyon_cc_library")
+load("//bazel:tachyon_cc.bzl", "tachyon_cc_library", "tachyon_cc_unittest")
 
 package(default_visibility = ["//visibility:public"])
 
+tachyon_cc_library(
+    name = "constraint_system",
+    hdrs = ["constraint_system.h"],
+    deps = [
+        ":variable",
+        "//tachyon/math/matrix:matrix_types",
+        "//tachyon/zk/air/expressions:air_evaluator",
+        "@eigen_archive//:eigen3",
```

Comment:
This is minor, though. This seems redundant, cause `//tachyon/math/matrix:matrix_types` already depends on this.